allennlp/data/dataset_readers/dataset_reader.py

```diff
@@ -163,6 +164,7 @@ def __init__(
             os.makedirs(self._cache_directory, exist_ok=True)
         self.manual_distributed_sharding = manual_distributed_sharding
         self.manual_multi_process_sharding = manual_multi_process_sharding
+        self._serialization_dir = serialization_dir
```

Comment:
Let's make this consistent with the other attributes:

```suggestion
        self.serialization_dir = serialization_dir
```